src/Admin/Model/UserAdmin.php

```diff
@@ -251,4 +251,16 @@ protected function configureFormFields(FormMapper $formMapper): void
             ->end()
         ;
     }
+
+    protected function isAllowedToSwitch(): bool
```

Comment:
No need to make this `protected`

Reply:
`isAllowedToSwitch` method will only be used in `UserAdmin` class.
No need to be `public`.
So i make it `private` ?

Reply:
You should always use the lowest visibility, if possible